src/js/movies.js

```diff
@@ -1,3 +1,87 @@
+import  movies  from   '../data/movies.json' assert { type: 'json' };
+
+console.log('movies2', movies) 
+
+let moviesList = document.querySelector('.movies-now')
+let futureMovies = document.querySelector('.movies-later ')
+
+
+class Movies {
+    constructor(title,description,country,languages,director, actors, posterUrl,imageUrl,videoUrl,genres,ageRestriction,releaseDate,screeningTimes,tags,rating){
+        this.title = title;
+        this.description = description;
+        this.country=country;
+        this.languages = languages;
+        this.director = director;
+        this.actors = actors;
+        this.posterUrl = posterUrl;
+        this.imageUrl = imageUrl;
+        this.videoUrl = videoUrl;
+        this.genres = genres;
+        this.ageRestriction = ageRestriction;
+        this.releaseDate = releaseDate;
+        this.screeningTimes = screeningTimes;
+        this.tags = tags;
+        this.rating = rating;     
+
+    }
```

Comment:
A thought regarding this constructor. The object/class Movies is as far I can see not created anywhere in the file. I see you used "static" on the functions so you don't need to create objects before using them :) So I think there's no need for the constructor in this case! What do you think? :P

Reply:
I only created the constructor because it will be used for other functions in the future

Reply:
Oh nice I see!
An alternative solution for how many parameters the constructor takes is to only take a "movie" object as parameter. Then you only need to set: this.movie = movie; and then you can use all the variables with e.g: movie.title, movie.rating.

Overall it looks good and easy to read! :)

Reply:
Ok thanks for the suggestion